third_party/proton/README.md

```diff
@@ -52,7 +52,32 @@ proton.activate(session_id)
 # Write out the profile data and finalize the profiler
 proton.finalize()
 ```
+### Advanced features
```

Comment:
I think advanced features can be moved to a later section, probably before or after "Instruction sampling".